packages/wrap/src/transforms/TransformInputObjectFields.ts

```diff
@@ -96,11 +96,18 @@ export default class TransformInputObjectFields implements Transform {
       if (variableDefs != null) {
         for (const variableDef of variableDefs) {
           const varName = variableDef.variable.name.value;
-          if (variableDef.type.kind !== Kind.NAMED_TYPE) {
-            throw new Error(`Expected ${variableDef.type} to be a named type`);
```

Comment:
It looks like this is the right direction, but looks like even older version would be better with a cast to NamedTypeNode as the switch statement is just calling the same code?

Reply:
Note the comment below. It seems to reference a cast that is no longer performed, probably replaced by error when we switched to strict mode?

Reply:
I improved the code